master/static/srv/tensorboard-entrypoint.sh

```diff
@@ -11,9 +11,7 @@ STARTUP_HOOK=${STARTUP_HOOK_SCRIPT:-startup-hook.sh}
 # Source `startup-hook.sh` before running tensorboard.
 source <(python3.6 -m determined.exec.prepare_env ${STARTUP_HOOK})
 
-# S3 backed experiments must have AWS_REGION set in the environment.
-eval "$(python3.6 -m determined.exec.tensorboard s3)"
-
-python3.6 -m determined.exec.tensorboard service_ready &
-
-tensorboard --port=${TENSORBOARD_PORT} --path_prefix="/proxy/${DET_TASK_ID}" $@
+python3.6 -m determined.exec.tensorboard \
+    --port="${TENSORBOARD_PORT}" \
+    --path_prefix="/proxy/${DET_TASK_ID}" \
```

Comment:
non-blocking: the code works as is but we can use this argument to build the URL for the service ready check. Right now we're checking this environment variable again to build the URL.

Reply:
I'll just move the args to inside the python code?

Reply:
Yup looks good!